pkg/util/validation_test.go

```diff
@@ -3,19 +3,15 @@ package util
 import (
 	"testing"
 
-	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestIsEmail(t *testing.T) {
-	Convey("When validating a string that is a valid email", t, func() {
-		result := IsEmail("abc@def.com")
-
-		So(result, ShouldEqual, true)
-	})
-
-	Convey("When validating a string that is not a valid email", t, func() {
-		result := IsEmail("abcdef.com")
-
-		So(result, ShouldEqual, false)
-	})
+	emails := map[string]bool{
+		"abc@def.com": true,
+		"abcdef.com":  false,
+	}
+	for input, expected := range emails {
+		assert.Equal(t, expected, IsEmail(input))
```

Comment:
I would like to keep the test descriptions; I would suggest wrapping in a `t.Run` block

Reply:
Agree with using `t.Run`.

Reply:
I added back some test descriptions, they're not _exactly_ as they were (mostly because GoConvey neatly shows descriptions and the Go test runner insists on formatting them with `_` instead of ` `).

I also added a few more test cases here because it felt relevant.